src/MockClassifier.Api/MockClassifier.Api.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-    <Nullable>enable</Nullable>
+    <Nullable>disable</Nullable>
```

Comment:
Fair enough - we're not using it.

Reply:
I consciously decided to switch this off just in case we use any code dependencies that may not have recompiled their binaries with this setting on. It could cause us problems.